src/device-manager/cocoa/NLWeaveLogging.mm

```diff
@@ -26,23 +26,24 @@
 #include <stdio.h>
 
 #import <Foundation/Foundation.h>
+#import "NLWeaveLogWriter.h"
+#import "NLWeaveLogging.h"
 
 #if TARGET_OS_IPHONE
 #import <UIKit/UIKit.h>
 #import <os/log.h>
 #endif
 
-#include <Weave/Support/logging/WeaveLogging.h>
+#include "third_party/weave/src/lib/support/logging/WeaveLogging.h"
```

Comment:
please use Weave/Support/logging/WeaveLogging.h instead of third_party/weave/src/lib/support/logging/WeaveLogging.h

Reply:
Done, thank you!